build/build.sh

```diff
@@ -9,6 +9,7 @@ echo "Retrieving and processing reference metadata"
 
 # pandoc settings
 CSL_PATH=references/style.csl
+DOCX_PATH=references/reference-docx.docx
```

Comment:
Can we change this filename to `pandoc-reference.docx`? I think that will make it a little more clear what it's purpose is.